app/controllers/results_controller.rb

```diff
@@ -166,11 +160,22 @@ def show
                                  peer_visible: is_review)
           criteria_info = criteria.pluck_to_hash(*fields)
           marks_info = criteria.joins(:marks)
-                               .where('marks.result_id': result.id)
-                               .pluck_to_hash(*fields, 'marks.mark')
-                               .group_by { |h| h[:id] }
-          criteria_info.map do |h|
-            info = marks_info[h[:id]]&.first || h.merge('marks.mark': nil)
+                              .where('marks.result_id': result.id)
```

Comment:
You should indent this line and the next three by 1 space (the diff is saying you changed these lines, but you didn't really, just the indentation is off).

Reply:
@ashih2018 it looks like the indentation is still off here. You need to align the periods with the line above.